specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2021-05-01/examples/CapabilitiesByLocationList.json

```diff
@@ -0,0 +1,1159 @@
+{
+  "parameters": {
+    "api-version": "2020-07-01-preview",
```

Comment:
>2020-07-01-preview [](http://example.com/codeflow?start=20&length=18)

There are multiple files with 2020-07-01-preview version. Please fix them all.

Reply:
Fixed in new commit